### DISCUSSION TITLE
ILS4L at KDTW

### DISCUSSION THREAD

**Discussion by Sky173** [flybywiresim/aircraft#4272] [Help] [answered]
Runway 4L at Detroit doesn't line up correctly. It appears to be lined up for the taxiway instead. I mentioned this in the MSFS forums but I was told this is specific to the aircraft, not the world.

Additionally, there is no "bug report" category here on GitHub, so I'm not sure where to post this. Thanks.

https://www.youtube.com/watch?v=ViYHXOJMsXI

**Comment by hotshotp** [comment, marked as answer]
there is a bug report category in this github, its the issues section. however this is not an a32nx issue but in fact an MSFS issue.
